### ISSUE TITLE
Use form for the demo

### ISSUE THREAD

**Issue by iMADi-ARCH** [joschan21/profanity.dev#1]
Use form for the demo so as to enable `checkProfanity` mutation to be run when the user presses the `enter` key.

**Comment by PrivFelixKniest** [comment]
No necessary need for a form here probably, you could also just catch the enter action and submit the Function that way, though I believe that this change would not provide a ton of value for the algorithm itself

**Comment by iMADi-ARCH** [comment]
@PrivFelixKniest yes. This feature is necessary but still counts as UX improvement. Using form is more accessible and will still work if there are more fields added in the future. 

I am still exploring the algorithm. I felt like this is a nice feature to have and took hardly 5 minutes to implement so I opened a PR.

**Comment by PrivFelixKniest** [comment]
yup, I can look over the PR aswell if you want to tag me, I guess josh has to review it anyways though

**Comment by PrivFelixKniest** [comment]
Ah wait yeah I found it
